modules/botAdmin.js

```diff
@@ -118,7 +118,7 @@ const Module = new Augur.Module()
       const client = msg.client;
 
       const embed = u.embed()
-      .setAuthor(client.user.username + " Heartbeat", client.user.displayAvatarURL())
+      .setAuthor({ name: client.user.username + " Heartbeat", iconURL: client.user.displayAvatarURL() })
```

Comment:
This is one of the few places that did need updating. Keep the change you made here (or even better, move the author into the call to u.embed() as is done in most other places).

Reply:
Since this one has a string attached, I've left it in the vanilla format